selftests/checkall

```diff
@@ -11,4 +11,3 @@ run_rc() {
 run_rc './avocado-static-checks/run-static-checks'
```

Comment:
```suggestion
run_rc './static-checks/run-static-checks'
```
I think that we need to update the path of `run-static-checks` since we have updated the configuration of the submodules `avocado-static-checks`: https://github.com/avocado-framework/avocado-vt/pull/4219/changes#diff-fe7afb5c9c916e521401d3fcfb4277d5071798c3baf83baf11d6071742823584
Please correct me if I was misunderstood, thanks.

BTW, could you elaborate more on why you update the submodule `avocado-static-checks`? Thanks.